include/base/libmesh_common.h

```diff
@@ -488,7 +405,7 @@ struct casting_compare {
 #define libmesh_exceptionless_error_msg(msg)                            \
   do {                                                                  \
     libMesh::Threads::lock_singleton_spin_mutex();                      \
-    libMesh::err << msg << std::endl;                                   \
+    libMesh::err << msg << '\n';                                        \
```

Comment:
Why is this change necessary? (Answer will apply to following occurrences)

Reply:
Not necessary (we're about to die, a few redundant buffer flushes aren't going to kill anything), I guess I just noticed that at this point we're already about to call `report_error()` which is always going to call `here()` which is always going to output `std::endl` anyway, so no reason not to wait until then to flush.